events/event-manager.js

```diff
@@ -0,0 +1,55 @@
+var eventModels = require('./event-models.js');
+var Event = eventModels.Event;
+// This function return an object containing the event UID of the newly created event.
+var createEvent = function(reqData, locals, createCallback) {
+  var event = new Event({
+    category: reqData.category,
+    contact: reqData.contact ,
+    creator: reqData.creator,
+    datetime: new Date(reqData.datetime),
+    description: reqData.description,
+    location: reqData.location,
+    title: reqData.title
+  });
+
+  event.save(createCallback)
+
+  return {
+    "eventUid": event.id
+  };
```

Comment:
The event UID is passed to the ```createCallback``` as part of the created object, so you just need to ```return;``` here.